rollouts/config/crd/bases/containercluster.yaml

```diff
@@ -0,0 +1,1336 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    cnrm.cloud.google.com/managed-by-kcc: "true"
+    cnrm.cloud.google.com/stability-level: stable
+    cnrm.cloud.google.com/system: "true"
+    cnrm.cloud.google.com/tf2crd: "true"
+    core.cnrm.cloud.google.com/configconnector: configconnector.core.cnrm.cloud.google.com
+  name: containerclusters.container.cnrm.cloud.google.com
```

Comment:
Does this file come from another Github source? We could consider using the remote URL in the kustomization in that case, instead of copying it over

Reply:
I want to find a better solution for this CRD. This is an external CRD (comes from KCC) and is already part of our release manifests, I will try to reuse that instead of adding a new one here.

Reply:
I am reusing the `containercluster.yaml` from the `manifests` directory for now. So we have one source of truth for now. Eventually we want to make `rollouts` to be able to gracefully handle unavailability of this CRD.